### PR TITLE
Add compatible conda

### PR DESCRIPTION
**Pull request by jdesanto**

### CODE REVIEW

File: notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb [hunk 1]
```diff
@@ -308,7 +315,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.8.13 ('adsdev')",
+   "display_name": "Python 3.11.1 64-bit",
```
Comment (darenr) [inline notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb:318]: this doesn't seem right? 3.11?

File: notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb [hunk 2]
```diff
@@ -322,11 +329,11 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.13"
+   "version": "3.11.1"
```
Comment (darenr) [inline notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb:332]: same comment, 3.11?

File: notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "@notebook{load_data-object_storage-hive-autonomous-database.ipynb,\n",
     "    title: Loading Data with DatasetFactory,\n",
```
Comment (darenr) [inline notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb:8]: This title is misleading, it's not showing that at all, it's (correctly) using the `pandas` accessor. The only use of DSF is `from_dataframe` to invoke `show_in_notebook()`, can you update the title please to reflect what it's really showing.

File: notebook_examples/streaming-service-introduction.ipynb [hunk 4]
```diff
@@ -47,6 +47,7 @@
     "This notebook uses the `kafka-python` client to connect to the Streaming service by leveraging the Streaming kafka-compatible API. Messages are consumed and produced using the `kafka-python` client.\n",
     "\n",
     "Developed on [Data Exploration and Manipulation](https://docs.oracle.com/en-us/iaas/data-science/using/conda-dem-fam.htm) for CPU on Python 3.7 (version 3.0)\n",
```
Comment (darenr) [inline notebook_examples/streaming-service-introduction.ipynb:49]: just to check, this really is still a 3.7 notebook right?

File: notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb [hunk 5]
```diff
@@ -40,6 +40,13 @@
     "---\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compatible conda pack: [General Machine Learning](https://docs.oracle.com/en-us/iaas/data-science/using/conda-gml-fam.htm) for CPU on Python 3.8 (version 1.0)"
```
Comment (darenr) [inline notebook_examples/load_data-object_storage-hive-autonomous-database.ipynb:47]: this link (https://docs.oracle.com/en-us/iaas/data-science/using/conda-gml-fam.htm) takes you not to a 3.8 conda but only 3.7 - seems like the wrong link

should be: 

https://docs.oracle.com/en-us/iaas/data-science/using/conda-generalml-fam.htm

**Comment by darenr** [comment]
@jdesanto closing this should happen when it's merged. You need to fix the review comments, then mark them as resolved along with the updated files. Then I can merge, once you've completed the OCA requirements that Ming shared.

File: notebook_examples/streaming-service-introduction.ipynb (hunk 4 above)
Comment (jdesanto) [reply notebook_examples/streaming-service-introduction.ipynb:49]: Yes

**Pull request merged**